data-raw/watershed/battle/battle_creek.rmd

```diff
@@ -16,20 +15,16 @@ library(cvpiaHabitat)
 None Planned
 
 ## Instream Spawning and Rearing Habitat
+**Data Source:** [Thomas R. Payne & Associates 1995](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/Payne1995_BattleCreekIFIM.pdf){target="_blank"} (pg C-1)
 
-**Data Source:**   
-[Thomas R. Payne & Associates 1995](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/Payne1995_BattleCreekIFIM.pdf) (pg C-1)
-
-Instream spawning and rearing habitat for Fall Run Chinook Salmon and Steelhead in Battle Creek is based on data from Thomas R. Payne & Associates instream flow evaluations conducted for the California Department of Fish and Wildlife. These evaluations developed relationships between flow and suitable spawning and rearing habitat for Battle Creek from seven study reaches in Battle Creek.  The consolidated WUA for all mainstem reaches are provided in Appendix C page C-1. [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service provided the consolidated full-river data for the entire Battle Creek watershed in a spreadsheet (see ‘~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx’), and instructed us to use A1:E36 of the ‘Battle’ tab within his ‘IFIMWUA.xlsx’. We consolidated aditional Steelhead data from Thomas Payne and Associates 1995 in a spreadsheet (see'~cvpiaHabitat/data-raw/weighted_composite_battle.csv).   
+Instream spawning and rearing habitat for Fall Run Chinook Salmon and Steelhead in Battle Creek are based on data from Thomas R. Payne & Associates instream flow evaluations conducted for the California Department of Fish and Wildlife. These evaluations developed relationships between flow and suitable spawning and rearing habitat for Battle Creek from seven study reaches. The consolidated WUA for all mainstem reaches is provided in Appendix C page C-1. [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service provided the consolidated full-river data for the entire Battle Creek watershed in a spreadsheet (see ‘~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx’), and instructed us to use A1:E36 of the ‘Battle’ tab within his ‘IFIMWUA.xlsx’. We consolidated additional Steelhead data from Thomas Payne and Associates 1995 in a spreadsheet (see'~cvpiaHabitat/data-raw/weighted_composite_battle.csv).   
```

Comment:
It doesn't look like you are using data from mark gard's spreadsheet at all, plz update text to reflect this